financial-services-accelerator/internal-apis/org.wso2.financial.services.accelerator.authentication.endpoint/src/main/webapp/includes/footer.jsp

```diff
@@ -0,0 +1,27 @@
+<%--
+ ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ ~
+ ~ WSO2 LLC. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied. See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ --%>
+
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<footer class="footer">
+    <div class="container-fluid">
+        <p>WSO2 Open Banking <script>document.write(new Date().getFullYear());</script>
```

Comment:
here as well

Reply:
Still we have not decided to change the naming open banking from the webapps